apps/daffio/src/app/design/device/laptop/laptop.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'daffio-laptop',
+  templateUrl: './laptop.component.html',
+  styleUrls: ['./laptop.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
I don't think you need the `ViewEncapsulation.None`

Reply:
Resolved in 07839c0